src/main/groovy/edu/oregonstate/mist/mealcard/mapper/MealPlanBalanceMapper.groovy

```diff
@@ -0,0 +1,37 @@
+package edu.oregonstate.mist.mealcard.mapper
+
+import edu.oregonstate.mist.mealcard.core.MealPlanBalance
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+class MealPlanBalanceMapper implements ResultSetMapper<MealPlanBalance> {
+
+    private static DateTimeFormatter dbFormatter = DateTimeFormatter
+            .ofPattern("yyyy-MM-dd HH:mm:ss")
+            .withZone(ZoneId.of("UTC"))
+
+    private static DateTimeFormatter outputFormatter = DateTimeFormatter
+            .ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'")
+            .withZone(ZoneId.of("UTC"))
+
+    @Override
+    MealPlanBalance map(int index , ResultSet rs , StatementContext ctx)
+            throws SQLException {
+        def dbDate = ZonedDateTime.parse(rs.getString("LAST_USED_DATE") ,dbFormatter)
```

Comment:
How about moving the date conversion to a separate method? Then line 33 could use that method. Let's also write a unit test for the date conversion method.